regulations/tests/templatetags_macros_tests.py

```diff
@@ -0,0 +1,38 @@
+from unittest import TestCase
+
+from django.template import Context, Template
+
+
+class MacrosTests(TestCase):
+    def _generate(self, content):
+        """Shorthand for passing the content into a template and rendering"""
+        text = "{% load macros %}" + content
+        return Template(text).render(Context({}))
+
+    def test_external_link_no_optional(self):
+        """The classes and title fields are optional. We should generate an
+        appropriate link"""
+        result = self._generate(
+            '{% external_link url="http://example.com/path" text="Click" %}')
+        self.assertTrue('<a' in result)
```

Comment:
For both these tests, wouldn't it be cleaner to have one assert against the expect `<a>` element?